README.md

```diff
@@ -39,13 +39,24 @@ def mystery2(n)
 end
 ```
 
-- What is mystery2(123)?
-- What is mystery2(9005)?
-- What is mystery2(-123)?
+- What is mystery2(123)? 6
+- What is mystery2(9005)? 14
+- What is mystery2(-123)? -123
 - _Added Fun: How could we make `mystery2(-123)` work the way we might expect it to work instead of the way it does?_
+  + This probably isn't the most efficient way, but here's what I got to work (returns -6 for mystery2(-123) and 6 for mystery2(123)):
```

Comment:
Heh - this is definitely a little confusing to read. But I did plug in some numbers and it seems like it works. Here's another way you could have tackled it which is maybe more easy to read:

```ruby
def mystery2(n)
  if n < 0
    return -1 * mystery2(-n)
  elsif n < 10
    return n
  else
    return (n%10) + mystery2(n/10)
  end
end
```